papers/copper-mountain-2021/lfa_pmultigrid_high_order_fem.tex

```diff
@@ -0,0 +1,973 @@
+The nodes $\mathbf{x}_{j, c}$ and are $\mathbf{x}_{i, f}$ are on the coarse and fine grids, respectively.
+\end{definition}
+
+\begin{definition}\label{def:restriction_symbol}
+The symbol of p-restriction is given by the expression
+\begin{equation}
+\tilde{\mathbf{R}}_{\text{ftoc}} \left( \boldsymbol{\theta} \right) = \mathbf{Q}_c^T \left( \mathbf{R}_e \odot \left[ e^{\imath \left( \mathbf{x}_{j, f} - \mathbf{x}_{i, c} \right) \cdot \boldsymbol{\theta} / \mathbf{h}} \right] \right) \mathbf{Q}_f,
+\end{equation}
+where $i \in \lbrace 1, 2, \dots, n \cdot \left( p_{\text{coarse}} + 1 \right)^d \rbrace$, $\mathbf{h}$ is the length of the element in each dimension, $j \in \lbrace 1, 2, \dots, n \cdot \left( p_{\text{fine}} + 1 \right)^d \rbrace$, $n$ is the number of components, $p_{\text{fine}}$ and $p_{\text{coarse}}$ are the polynomial orders of the fine and coarse grid discretizations, respectively, and $d$ is the dimension of the finite element basis.
+The matrices $\mathbf{Q}_f$ and $\mathbf{Q}_c$ are the localization mappings for the fine and coarse grid, respectively, and the element p-restriction operator is given by $\mathbf{R}_e = \mathbf{P}_e^T = \mathbf{B}_{\text{ctof}}^T \mathbf{D}_{\text{scale}}$.
+The nodes $\mathbf{x}_{j, f}$ and are $\mathbf{x}_{i, c}$ are on the fine and coarse grids, respectively.
+\end{definition}
+
+% -----------------------------------------------------------------------------
+\subsubsection{Multigrid Error Propagation Symbol}\label{sec:multigridsymbol}
+% -----------------------------------------------------------------------------
+
+Based on the previous discussion, we can combine the expressions in p-multigrid to give the symbol of the error propagation operator.
+
+\begin{definition}\label{def:pmultigrid_symbol}
+The symbol for the p-multigrid error propagation operator is given by
+\begin{equation}
+\tilde{\mathbf{E}}_{\text{2mg}} \left( \nu, \omega, \boldsymbol{\theta} \right) = \tilde{\mathbf{S}}_f \left( \nu, \omega, \boldsymbol{\theta} \right) \left[ \mathbf{I} - \tilde{\mathbf{P}}_{\text{ctof}} \left( \boldsymbol{\theta} \right) \tilde{\mathbf{A}}_c^{-1} \left( \boldsymbol{\theta} \right) \tilde{\mathbf{R}}_{\text{ftoc}} \left( \boldsymbol{\theta} \right) \tilde{\mathbf{A}}_f \left( \boldsymbol{\theta} \right) \right] \tilde{\mathbf{S}}_f \left( \nu, \omega, \boldsymbol{\theta} \right),
+\end{equation}
+where $\tilde{\mathbf{A}}$ is the symbol of the fine grid operator, $\tilde{\mathbf{S}}_f \left( \nu, \omega, \boldsymbol{\theta} \right)$ is the symbol of the smoother and $\tilde{\mathbf{A}}_c^{-1} \left( \boldsymbol{\theta} \right)$ is the symbol of the coarse grid solve.
+$\tilde{\mathbf{P}}_{\text{ctof}} \left( \boldsymbol{\theta} \right)$ and $\tilde{\mathbf{R}}_{\text{ftoc}} \left( \boldsymbol{\theta} \right)$ represent the symbol of the grid prolongation and restriction operators, respectively.
```

Comment:
I specifically included this sentence to make the definition more self contained.